src/pages/en/core-concepts/framework-components.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Framework Components
+description: Learn how to use React, Svelte, etc.
+---
+Astro allows you to build your site using JavaScript UI components from a variety of popular frameworks. Astro will run your [React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) and [Lit](https://lit.dev/) components at build time, then render your entire site to static HTML by default, removing all the JavaScript from the final page!
+
+```astro
+---
+import MyReactComponent from '../components/MyReactComponent.jsx';
+---
+<html>
+  <head>
+    <title>Look How Easy it is to Use a React Component!</title>
+  </head>
+  <body>
+    <h1>My Astro Page</h1>
+    <MyReactComponent />
+  </body>
+</html>
+```
+
+You can use any of these frameworks as templating languages to build static pages, taking advantage of the syntax and tooling you already know and love. This allows you to build with components you're familiar with, maybe even the exact same ones you've already written! And, you won't send any unnecessary JavaScript to the client to render static elements, even dynamically generated ones.
+
+For client-side interactivity, Astro allows you to opt-in to load a single component's JavaScript and dpendencies only when and where that component is used.
+
+> 📚 Read more about Astro's [partial hydration](/en/core-concepts/partial-hydration)
+
+## Configure Framework Renderers
+
+Your Astro configuration must include the renderer for each framework you use. You can find, and manually edit, your project's current list of available renderers in `astro.config.mjs`.
+
+```js
+export default ({
+// No renderers are needed for AlpineJS support, just use Astro components!
+renderers: [
+    '@astrojs/renderer-svelte',
+    '@astrojs/renderer-vue',
+    '@astrojs/renderer-react',
+    '@astrojs/renderer-preact',
+    '@astrojs/renderer-lit',
+    '@astrojs/renderer-solid',
+  ],
+
+});
+```
+
+⚙️ See the [Renderer Reference](/en/reference/renderer-reference) to learn about configuring custom renderers.
+
+## Build With Components
+
+Use your JavaScript framework components in your Astro pages, layouts and components just like Astro components! All your components can live together in `/src/components`, or can be organized in any way you like.
+
+### Render on a Page
+
+To use a framework component, import it from its relative path, including file extension, in the component script. Then, use the component alongside other components, HTML elements and JSX-like expressions in the component template. 
+```astro
+---
+import BaseLayout from '../layouts/BaseLayout.astro';
+import MyReactComponent from '../components/MyReactComponent.jsx';
+import MySvelteComponent from '../components/MySvelteComponent.svelte';
```

Comment:
woah, are you using multiple components on the same page??? Oh boy!!! That seems really cool and special, probably worth introducing in its own section, otherwise this could feel like an overwhelming first example. 


Given all of the above, how about a reorg like this?

```diff
- ## Build With Components
- ### Render on a Page
+ ## Static Framework Components
+ ## Hydrating Interactive Components
+ ## Nesting Components
+ ## Mixing Multiple Frameworks
```

Reply:
resolved in "freds changes"